app/overlays/strip_elf.conf

```diff
@@ -0,0 +1,7 @@
+# The additional stripped .elf files are deterministic.
+# This adds very few files in the build directory and has
+# absolutely no other effect whatsoever.
+# See on-going discussion in
+# https://github.com/zephyrproject-rtos/zephyr/pull/51954
+# and usage in xtensa-build-zephyr.py
+CONFIG_BUILD_OUTPUT_STRIPPED=y
```

Comment:
btw is there any reason why this is done in separate overlay?
If you want to simply turn it on by default for SOF+Zephyr please check prj.conf for such cases:
https://github.com/thesofproject/sof/blob/main/app/prj.conf
so there will be no need to modify build script for single config

Reply:
> btw is there any reason why this is done in separate overlay?
If you want to simply turn it on by default for SOF+Zephyr please check prj.conf for such cases:

No reason, great suggestion. Done, thanks!